test/attributes/spaceAfterAt.chpl

```diff
@@ -0,0 +1 @@
+@ chpldoc.nodoc proc foo() { }
```

Comment:
I'd also check what happens when the attribute doesn't have an associated namespace (e.g., `@ deprecated`).  The chpldoc case was the side I was more worried about, though, so I don't expect any surprises